### ISSUE TITLE
Retreive Unspent outputs from Bitcoin Core REST API

### ISSUE THREAD

**Issue by lclc** [OutCast3k/coinbin#27]
Would be cool if one can get the UTXO from the Bitcoin Core REST API instead of using a 3rd party service:
https://github.com/bitcoin/bitcoin/blob/master/doc/REST-interface.md#query-utxo-set

Ones sendrawtransaction is implemented in the Bitcoin Core REST API you could run coinb.in with your own full node (or anyones full node that provides the REST API to the public).

**Comment by lclc** [comment]
See this Bitcoin Core PR regarding sendrawtransaction: https://github.com/bitcoin/bitcoin/pull/6844

**Comment by omefire** [comment]
+1

**Comment by OutCast3k** [comment]
Yeah, I like this idea... It will be something, I'll work on very very soon.

**Comment by ninate** [comment]
Sounds great

**Comment by ninate** [comment]
[nicole-master.zip](https://github.com/OutCast3k/coinbin/files/4007048/nicole-master.zip)
